kaggle/api/kaggle_api_extended.py

```diff
@@ -1044,10 +1044,12 @@ def dataset_metadata(self, dataset, path):
 
         result = self.process_response(
             self.metadata_get_with_http_info(owner_slug, dataset_slug))
-        if (result['errorMessage']):
+        print(result)
+        print(type(result))

```

Comment:
```suggestion
        print(result)
        print(type(result))

```
let's drop this debugging code...